automated install/basic-install.sh

```diff
@@ -357,7 +357,7 @@ package_manager_detect() {
         # These variable names match the ones for apt-get. See above for an explanation of what they are for.
         PKG_INSTALL=("${PKG_MANAGER}" install -y)
         # CentOS package manager returns 100 when there are packages to update so we need to || true to prevent the script from exiting.
-        PKG_COUNT="${PKG_MANAGER} check-update | grep -E '(.i686|.x86|.noarch|.arm|.src)' | wc -l || true"
+        PKG_COUNT="${PKG_MANAGER} check-update | grep -E '(.i686|.x86|.noarch|.arm|.src|.riscv64)' | wc -l || true"
```

Comment:
This should be verified by someone with the hardware running an RPM distro.

Reply:
I'm not sure this will happen any time soon. As nobody replied so far, I'm okay with merging this now and instead waiting to see if someone complains later. The RISC-V 64bit is a testing environment that doesn't run smoothly everywhere at this point so this seems acceptable. Nobody should be using it for critical/important services without having a fallback solution right at hand.

Reply:
I hope to receive a PINE64 Star64 soon which allows me do test this.